test/esinstall/auto-named-exports/packages/cjs-named-export-pkg-02/entrypoint.js

```diff
@@ -0,0 +1,2 @@
+exports.export1 = 'foo';
+exports.export2 = 'bar';
```

Comment:
Not blocking this PR, but it’d be great to test `module.exports = [various things that may conflict / compete with the other exports]` either in the same package or another package

Reply:
since we're falling back to https://github.com/guybedford/cjs-module-lexer for this analysis, we can assume the edge cases are already tested on their end. But, I agree we could do a bit more here than just this simple example, will add a few more lines/exports.